terraform/modules/s3_bucket/main.tf

```diff
@@ -0,0 +1,121 @@
+variable "bcuket_name" {
```

Comment:
We could use this for the terraform state if we don't mind it being dependent on a module.